### DISCUSSION TITLE
WebUI

### DISCUSSION THREAD

**Discussion by ExperienceRO** [new-sankaku/stable-diffusion-webui-simple-manga-maker#35] [Q&A] [answered]
Hi... Yeah, me again...

So, as you must know, Forge is like changing to an experimental webui: https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/801

My question is if you are considering to move to mainstream WebUI A1111 or if Manga Maker will stay using Forge??

Domo..

**Comment by new-sankaku** [comment]
MangaMaker works with both WebUI and Forge, so there is no migration work to be done as a project. However, since development of Forge has ended, I think most users will migrate to WebUI.

However, there is one problem - WebUI uses a lot of VRAM, so I think we need to consider supporting ComfyUI as a project. However, since I don't use ComfyUI on a daily basis, I don't really understand the specifications. Especially the workflow configuration.

**Reply by ExperienceRO** [reply]
Oh, I see..
I thought Manga Maker only worked on Forge, and that's why I used Forge. I'm training my new SD3 model on A1111, and it seems to be better optimized, so I'll install Manga Maker there and try it out.
Thanks.

**Comment by new-sankaku** [comment, marked as answer]
By the way, you can launch it not only as an extension but also from the following URL:
https://new-sankaku.github.io/SP-MangaEditer/

Alternatively, if you want to launch it offline, you can download the following Git as is and double-click index.html to launch it locally.
https://github.com/new-sankaku/SP-MangaEditer
